irohad/ametsuchi/impl/postgres_wsv_command.hpp

```diff
@@ -26,40 +26,41 @@ namespace iroha {
     class PostgresWsvCommand : public WsvCommand {
      public:
       explicit PostgresWsvCommand(pqxx::nontransaction &transaction);
-      bool insertRole(const std::string &role_name) override;
+      WsvCommandResult insertRole(const std::string &role_name) override;
```

Comment:
naming is bad, IMO.

use just Result.

Reply:
I think it is better than Result<void, WsvError>, because when type is defined, we can clearly document why it is intended to be exactly that result

Reply:
Imo it's fine